### DISCUSSION TITLE
[Question]: Fix incorrectly submitted event to Frigate+?

### DISCUSSION THREAD

**Discussion by pyrodex** [blakeblackshear/frigate#10858] [Ask A Question]
### What is your question:

I thought I saw this before but can't find it via search...

Is there a way to adjust a submission to Frigate+ after hitting the submit to Frigate+ if you incorrectly flagged as NOT a false positive but it really was supposed to be?

**Comment by NickM-27** [comment]
No, there is no way to mark something as a false positive if it was not marked as such originally

**Comment by pyrodex** [comment]
Figured as much… Thanks!
